src/dotnet/commands/dotnet-help/LocalizableStrings.cs

```diff
@@ -28,34 +28,36 @@ internal class LocalizableStrings
 
         public const string Commands = "Commands";
 
-        public const string NewDefinition = "Initialize a basic .NET project";
+        public const string NewDefinition = "Initialize .NET projects.";
 
-        public const string RestoreDefinition = "Restore dependencies specified in the .NET project";
+        public const string RestoreDefinition = "Restore dependencies specified in the .NET project.";
 
-        public const string BuildDefinition = "Builds a .NET project";
+        public const string BuildDefinition = "Builds a .NET project.";
 
-        public const string PublishDefinition = "Publishes a .NET project for deployment (including the runtime)";
+        public const string PublishDefinition = "Publishes a .NET project for deployment (including the runtime).";
 
-        public const string RunDefinition = "Compiles and immediately executes a .NET project";
+        public const string RunDefinition = "Compiles and immediately executes a .NET project.";
 
-        public const string TestDefinition = "Runs unit tests using the test runner specified in the project";
+        public const string TestDefinition = "Runs unit tests using the test runner specified in the project.";
 
-        public const string PackDefinition = "Creates a NuGet package";
+        public const string PackDefinition = "Creates a NuGet package.";
 
-        public const string MigrateDefinition = "Migrates a project.json based project to a msbuild based project";
+        public const string MigrateDefinition = "Migrates a project.json based project to a msbuild based project.";
 
         public const string ProjectModificationCommands = "Project modification commands";
 
-        public const string AddDefinition = "Add items to the project";
+        public const string AddDefinition = "Add items to the project.";
         
-        public const string RemoveDefinition = "Remove items from the project";
+        public const string RemoveDefinition = "Remove items from the project.";
 
         public const string AdvancedCommands = "Advanced Commands";
 
-        public const string NugetDefinition = "Provides additional NuGet commands";
+        public const string NugetDefinition = "Provides additional NuGet commands.";
 
-        public const string MsBuildDefinition = "msbuilds a project and all of its dependencies";
+        public const string MsBuildDefinition = "Builds a project using MSBuild and all of its dependencies.";
```

Comment:
You're right, it is telling you nothing. In general, it should be "Allows running MSBuild'